tests/console/podman_image.pm

```diff
@@ -25,6 +25,17 @@ sub run {
 
     zypper_call "in podman";
 
+    # If there is no other CNI configured...
+    if (get_var("CNI", "podman") == "podman") {
+        # ... use the minimal podman CNI
+        zypper_call "in podman-cni-config";
+    }
+
+    if (is_leap("=15.1")) {
```

Comment:
So we wont need to install the parser on 15.2 or later?
What about SLE?

Reply:
According to the bug report it's stuck in SLE 15 SP1 maintenance.